pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -325,7 +325,7 @@ private void handleTransientStreamErrors(Exception e) throws Exception {
   protected boolean consumeLoop() throws Exception {
     _fieldExtractor.resetCounters();
     final long idlePipeSleepTimeMillis = 100;
-    final long maxIdleCountBeforeStatUpdate = (3 * 60 * 1000)/(idlePipeSleepTimeMillis + _streamMetadata.getKafkaFetchTimeoutMillis());  // 3 minute count
+    final long maxIdleCountBeforeStatUpdate = (3 * 60 * 1000)/(idlePipeSleepTimeMillis + _streamConfig.getKafkaFetchTimeoutMillis());  // 3 minute count
```

Comment:
I suppose you are going to change the methods in streamconfig to not say getKafkaFetchTimeout? Does that change make sense in this PR or in a later one? Since LLRealtimeSegmentDataManager is stream independent, we should not make kafka-specific calls from this class.

Reply:
I will make this change in the next couple of PRs, which will introduce stream specific StreamConfigs as wrappers around StreamConfig eg. 
`KakfaStreamConfig config = KafkaStreamConfig(streamConfig)`
As part of that, will go through all methods, and split them up into generic method calls vs specific calls. I was trying to avoid logic changes in this one.

Reply:
Perfect, thanks.